### ISSUE TITLE
[Payment] FromScript

### ISSUE THREAD

**Issue by altafan** [vulpemventures/go-elements#10]
A `FromScript` function similar to the existing `FromPublicKey` one must be added to the `payment` package in order to return a `Payment` from an output script and a network.

**Comment by altafan** [comment]
Taking into account the considerations pointed in https://github.com/vulpemventures/go-elements/issues/12#issuecomment-629326718, the `FromScript` method actually needs to return a _nested payment_.

This means that firstly the incoming script should be used to create a _redeem_ payment, then the _redeem_ would be passed to the `FromPayment` function.

Example:

```
func FromScript(script []byte) (*Payment, error) {
   redeem := &Payment{Script: script}
   return FromPayment(redeem)
}
```

In this way, we would be  sure that the returned payment has a non `nil` `Redeem` and that the `ScriptHash` and `WitnessScriptHash` function would return the correct addresses.

**Comment by tiero** [comment]
This should NOT be taken before the fix to be done described in #35
